### ISSUE TITLE
NamedNode IRI "username" must be absolute.

### ISSUE THREAD

**Issue by kristophor** [solid/node-solid-server#898]
when creating user on both inrupt and solid.community, i got this error loading the webpage.
it is preventing profile viewer and public folder viewer and no permission accessing inbox

**Comment by kjetilk** [comment]
I don't see any error message or attachment, perhaps you forgot to attach it?

**Comment by kristophor** [comment]
<img width="442" alt="screen shot 2018-11-02 at 10 34 01 am" src="https://user-images.githubusercontent.com/7058039/47921452-032e8180-de8b-11e8-8d1c-b22825874248.png">

**Comment by kristophor** [comment]
the issue seems to be intermittent, right now i have tried again on krischen.solid.community/ and it appears to be working again.

but it is not working for https://krischen.inrupt.net/

**Comment by kristophor** [comment]
maybe this is the data browser issue instead of solid server issue ?

**Comment by kjetilk** [comment]
Yeah, or possibly a solid-ui problem. I'll put the triage label on :-)
